src/views/Languages/bugs.graphql

```diff
@@ -0,0 +1,18 @@
+query Bugs($search: BugSearch!, $paging: Paging) {
+  bugs(search: $search, paging: $paging) {
+    edges {
+      node {
+        assignedTo {
+          name
+        }
+        component
+        lastChanged
+        id
+        keywords
+        summary
+        status
+        whiteboard
+      }
+    }
+  }
+}
```

Comment:
This seems to be a duplicate of `src/views/Project/bugs.graphql`. Move this file to `src/views` and import it from there for both views.